sklearn/impute.py

```diff
@@ -133,6 +133,10 @@ class SimpleImputer(BaseEstimator, TransformerMixin):
         - If X is not an array of floating values;
         - If X is encoded as a CSR matrix.
 
+    add_indicator : boolean, optional (default=False)
```

Comment:
When `copy=False` and `add_indicator=True`, a copy is still made. We should document it in the copy description.